src/config/constants.js

```diff
@@ -1,2 +1,3 @@
 /* eslint-disable import/prefer-default-export */
 export const BRAND_NAME = 'DreamScholars';
+export const EXPERIMENTAL_VERSION = true;
```

Comment:
Try this instead:
```suggestion
const params = new URLSearchParams(window.location.search);
expert const EXPERIMENT_SHOW_FULL_PROFILE_MENU = params.get("experimentShowFullProfileMenu") === "1";
```

And then when you access the site it'll be disabled by default unless you add `?experimentShowFullProfileMenu=1` to the URL.

Reply:
Ok I think the logic for how I currently has will need to be swapped meaning my current false will need to be changed to the true state for this to work properly but it does work just needs some edits ill figure it out